changelog/Microsoft.AAD.Reporting.json

```
@@ -1,6 +1,216 @@
+          "ApiChange": "Resource",
+          "ChangedApiName": "auditUserIdentity",
+          "ChangeType": "Addition",
+          "Description": "Added the [auditUserIdentity](https://docs.microsoft.com/en-us/graph/api/resources/auditUserIdentity?view=graph-rest-beta) resource type.",
+          "Target": "auditUserIdentity"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "Resource",
+          "ChangedApiName": "authenticationStrength",
+          "ChangeType": "Addition",
+          "Description": "Added the [authenticationStrength](https://docs.microsoft.com/en-us/graph/api/resources/authenticationStrength?view=graph-rest-beta) resource type.",
+          "Target": "authenticationStrength"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "Property",
+          "ChangedApiName": "alternateSignInName",
+          "ChangeType": "Deletion",
+          "Description": "Removed the **alternateSignInName** property from [signIn](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource.",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "Property",
+          "ChangedApiName": "authenticationProtocol",
+          "ChangeType": "Addition",
+          "Description": "Added the **authenticationProtocol** property to [signIn](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource.",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "String",
+          "ChangedApiName": "authenticationRequirement",
+          "ChangeType": "Addition",
+          "Description": "Added the [authenticationRequirement](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "String",
+          "ChangedApiName": "authenticationRequirement",
+          "ChangeType": "Deletion",
+          "Description": "Removed the **authenticationRequirement** resource type",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "String",
+          "ChangedApiName": "homeTenantId",
+          "ChangeType": "Deletion",
+          "Description": "Removed the **homeTenantId** resource type",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "String",
+          "ChangedApiName": "homeTenantId",
+          "ChangeType": "Addition",
+          "Description": "Added the [homeTenantId](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource",
+          "Target": "signIn"
+        },
```

Conflict (one object is removing another is adding) and wrong entries here - **homeTenantId** is a property of signIn
```suggestion
          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
          "ApiChange": "Property",
          "ChangedApiName": "homeTenantId",
          "ChangeType": "Addition",
          "Description": "Added the **homeTenantId** property to the [signIn](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource.",
          "Target": "signIn"
        },
```